README.md

```diff
@@ -8,6 +8,62 @@ It has the changes to the core schema.
 * It adds a packageMetadata section to releases/records.  This is because for an API, package metadata may be vary in each release/record and can not be applied to all results in an API. i.e there may be a variety of licenses in a single result.
 * It removes mandatory metadata fields from the release/record package as its is now possible (due to the previous change) for this data to be in the release/records themselves.
 
+## Example
+
+A publisher has a service to search releases by date, and his publication includes data from two different systems, one for tenders and other for contracts. As the result may include many releases, the publisher returns a paginated package as:
```

Comment:
The example is showing two things, but it's not clear from this paragraph. Maybe have one example for `packageMetadata`, and another for `links`, each with their own paragraph?